src/peripherals/atecc608a-tnglora-se/se-identity.h

```diff
@@ -127,11 +128,11 @@ extern "C" {
              */                                                                                                        \
             .KeyID         = NWK_KEY,                                                                                  \
             .KeySlotNumber = TNGLORA_APP_KEY_SLOT,                                                                     \
```

Comment:
Should be `.KeySlotNumber = TNGLORA_NWK_KEY_SLOT,`

Reply:
Yes, you are right.

Actually the AppKey and NwkKey slot is the same; maybe we should call it the root key slot and differentiate the block index for AppKey and NwkKey.

I'll follow up with a fix, thanks.